### ISSUE TITLE
Horizontal list AutoScrolls vertically

### ISSUE THREAD

**Issue by fierysolid** [computerjazz/react-native-draggable-flatlist#431]
**Describe the bug**
When dragging an item in a horizontal NestedDraggableList (with a NestableScrollContainer) to the edge of the screen to trigger auto-scroll, the screen scrolls vertically instead of horizontally

**To Reproduce**
Snack fails with: __reanimatedWorkletInit is not a function

**Platform & Dependencies**
 - react-native-draggable-flatlist version: 4.0.0
 - Platform: iOS 16
 - React Native: 0 68.5
 - Reanimated version: 2.13.0
 - React Native Gesture Handler version: 2.8.0

**Additional context**
https://user-images.githubusercontent.com/7339664/203354540-4c8f9601-1d2c-45c8-a1a0-36ebb99e0867.mov

**Comment by computerjazz** [comment]
mind posting the snack anyway? I've been having to patch in `global__reanimatedWorkletInit` in all my snacks, not sure what's up on their end -- see the `shims` file: https://snack.expo.dev/@computerjazz/draggable-flatlist-examples

**Comment by fierysolid** [comment]
Nice! I added that global directly to my file, and it didn't work, but the shim works great. Here's the snack:

https://snack.expo.dev/@fierysolid/mad-churros

Btw, I've since fixed this in my project by deconstructing the NestableScrollContainer in my code. Essentially the NestedScrollContainer doesn't know that its children are horizontal, so it still tries to scroll vertically. Also, the NestableDraggableFlatList has `scrollEnabled={false}` hardcoded because I think it is assumed the nested lists will always be vertical.

**Comment by computerjazz** [comment]
cool thanks! last question -- do you know if you actually need the `NestableScrollContainer`? The purpose of the container was to be able to autoscroll an outer container when using draggable lists in the same orientation as the container (eg. I have a vertically-scrollable menu with many sections that each contain a vertically-draggable list). 
I don't think this comes into play for you? Sounds like a good bug to squash nonetheless.

**Comment by fierysolid** [comment]
That was the first thing I tried actually. The gestures for the parent ScrollView get stolen by the FlatList children, so you need to use the ScrollContainerContext to pass information between them, so you know which one should be getting the swiping vertical vs horizontal swipe gestures.

**Comment by computerjazz** [comment]
Hm, I'm guessing the fact that the nestable version works is just a side effect of it having a default `activationDistance` prop, which is there for this purpose -- to prevent the children from stealing gestures: https://github.com/computerjazz/react-native-draggable-flatlist/blob/main/src/components/NestableDraggableFlatList.tsx#L92

I bet if you added `activationDistance` of `20` to the regular `DraggableFlatList` it would work just the same.

**Comment by fierysolid** [comment]
Yeah, that worked... wow so much effort with such an easy solution. Thank you
